@stellar/anchor-tests/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stellar/anchor-tests",
-  "version": "0.6.17",
```

Comment:
We never released 0.6.17 so I think it's ok if we don't need to bump the version.